src/tsickle_compiler_host.ts

```diff
@@ -30,56 +30,27 @@ export enum Pass {
   CLOSURIZE
 }
 
-export interface Options {
-  googmodule?: boolean;
-  es5Mode?: boolean;
-  prelude?: string;
-  /**
-   * If true, convert every type to the Closure {?} type, which means
-   * "don't check types".
-   */
-  untyped?: boolean;
-  /**
-   * If provided a function that logs an internal warning.
-   * These warnings are not actionable by an end user and should be hidden
-   * by default.
-   */
-  logWarning?: (warning: ts.Diagnostic) => void;
-  /** If provided, a set of paths whose types should always generate as {?}. */
-  typeBlackListPaths?: Set<string>;
-  /**
-   * Convert shorthand "/index" imports to full path (include the "/index").
-   * Annotation will be slower because every import must be resolved.
-   */
-  convertIndexImportShorthand?: boolean;
+export interface Options extends Es5ProcessorOptions, tsickle.AnnotatorOptions {
+  // This method is here for backwards compatibility.
+  // Use the method in TsickleHost instead.
+  logWarning?: TsickleHost['logWarning'];
```

Comment:
I think quoted accesses like this can break optimization.  Why not TsickleHost.logWarning?